app/javascript/react/components/capes/CapeShow.js

```diff
@@ -2,13 +2,13 @@ import React from "react"
 import { Link } from "react-router-dom"
 
 const CapeShow = ({ capeData }) => {
-  let { name, fullName, gender, affiliation, intelligence, strength, speed, url } = capeData
+  let { name, full_name, gender, affiliation, intelligence, strength, speed, url } = capeData
```

Comment:
This should in fact be set up to use the camelCase version `fullName` instead of the underscore `full_name`, because we are in a javascript file.  This can be handled in a serializer before being sent back up to the front end.